torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -948,6 +948,48 @@ static void fuseLogSoftmaxNllLoss(Block* b) {
   }
 }
 
+static void fuseSequenceSplitConcat(Block* b) {
```

Comment:
Is this fuse needed for export of loops?
Can you add comments to describe it?

Reply:
yes, export of loops sometimes creates (split -> concat) pair which is actually no-op. This optimization removes such cases if possible. Updated the name to `removeSequenceSplitConcat`, and added comments.